docs/prover/cli/options.md

```diff
@@ -243,7 +243,7 @@ When we do not care much for the output. It is recommended when running the tool
 
```

Comment:
There should probably be a note somewhere in the CLI options page that mentions that you can run with a conf file and links to the conf file page.

Reply:
done